content/riak/kv/2.1.4/using/cluster-operations/updating-search-schemas.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Updating Search Schemas"
+description: ""
+project: "riak_kv"
+project_version: "2.1.4"
+menu:
+  riak_kv-2.1.4:
+    name: "Updating Search Schemas"
+    identifier: "cluster_operations_update_search_schemas"
+    weight: 117
+    parent: "managing_cluster_operations"
+toc: true
+canonical_link: "https://docs.basho.com/riak/kv/latest/using/cluster-operations/updating-search-schemas"
+---
+
+The following page provides instruction on updating [search schemas](../../../developing/usage/search-schemas).
```

Comment:
It would be cool to maybe have a sentence or two here about when/why I'd want to update my search. Something like, "The following page provides instruction on updating [search schemas](../../../developing/usage/search-schemas), which is helpful when you need/when you want to..."
Maybe @angrycub or @jenagraham can weigh in with ideas here.